Source/SwipeCollectionViewCell.swift

```diff
@@ -133,6 +133,21 @@ open class SwipeCollectionViewCell: UICollectionViewCell {
             reset()
         }
     }
+
+    /// :nodoc:
+    override open func layoutSubviews() {
+        super.layoutSubviews()
+        if state.isActive {
```

Comment:
Could we improve open source solution and decrease copy pasting of this block?

For example, you can move this reset operation into swipeController and call it like `swipeController.reset(to: state)`

Reply:
Added `restoreActiveState` method